bench_mystr_test.go

```diff
@@ -0,0 +1,28 @@
+// Import StrToInt1 and do Benchmark tests
+package StrToInt2
+
+import (
+	"testing"
+     "StrToInt1" // go get StrToInt
```

Comment:
As a common practice benchmarks and tests stored in the same package as a code

Reply:
I've improved it all, is it better now?